pages/hub/apps/guides/debridge.mdx

```diff
@@ -0,0 +1,301 @@
+import { Callout } from 'components'
+
+# deBridge cross-chain bets
+
+The [deBridge](https://docs.dln.trade/dln-api/quick-start-guide) provides the way to execute user's bet transaction in `Polygon` from another blockchain, for example `Arbitrum`.
```

Comment:
The [deBridge API](https://docs.dln.trade/dln-api/quick-start-guide) allows to execute bets on `Polygon` originating from different blockchains, such as `Arbitrum`.

Reply:
f13e4127dec44bcbf66bbd1bc0b1418ef32f9a51